appsTransfer/src/main/java/org/kinecosystem/appstransfer/view/AppsTransferActivity.kt

```diff
@@ -14,18 +16,25 @@ import org.kinecosystem.appstransfer.presenter.AppsTransferPresenter
 import org.kinecosystem.appstransfer.view.customview.AppsTransferList
 import org.kinecosystem.common.utils.navigateToUrl
 import org.kinecosystem.transfer.model.EcosystemApp
+import org.kinecosystem.transfer.repositories.EcosystemAppsLocalRepo
+import org.kinecosystem.transfer.repositories.EcosystemAppsRemoteRepo
+import org.kinecosystem.transfer.repositories.EcosystemAppsRepository
+import org.kinecosystem.transfer.sender.manager.TransferManager
 
 class AppsTransferActivity : AppCompatActivity(), IAppsTransferView {
 
-
     private lateinit var dataGroup: Group
     private lateinit var noDataGroup: Group
     private lateinit var loader: ProgressBar
     private lateinit var list: AppsTransferList
     private var presenter: AppsTransferPresenter? = null
 
-    override fun transferToApp(app: EcosystemApp) {
-         Log.d("####", "#### AppsTransferActivity  start transferToApp")
+    override fun onTransferError() {
```

Comment:
add an error message and / or exception to the method

Reply:
will be done other pr